vaadin-dialog-flow-parent/vaadin-dialog-flow/src/main/java/com/vaadin/flow/component/dialog/Dialog.java

```diff
@@ -621,9 +624,47 @@ public Registration addDetachListener(
         return super.addDetachListener(listener);
     }
 
+
+    /**
+     * Adds theme variants to the component.
+     *
+     * @param variants
+     *            theme variants to add
+     */
+    public void addThemeVariants(DialogVariant... variants) {
+        getThemeNames()
+                .addAll(Stream.of(variants)
+                        .map(DialogVariant::getVariantName)
+                        .collect(Collectors.toList()));
+    }
+
+    /**
+     * Removes theme variants from the component.
+     *
+     * @param variants
+     *            theme variants to remove
+     */
+    public void removeThemeVariants(DialogVariant... variants) {
+        getThemeNames()
+                .removeAll(Stream.of(variants)
+                        .map(DialogVariant::getVariantName)
+                        .collect(Collectors.toList()));
+    }
+
```

Comment:
This code is related to another feature not intended for this version of flow. Please remove.

Reply:
Fixed